packages/client/src/components/Footer/Footer.style.css

```diff
@@ -1,176 +1,223 @@
+@import url('https://fonts.googleapis.com/css?family=Open+Sans');
+
+footer,
+footer > * {
+  box-sizing: border-box;
+}
+
+footer,
+.footer-bg,
 .footer-container,
-.footer-content-section,
-.footer-copyright-section,
-.placeholder-text {
+.copyright-disclaimer-container,
+.find-information-container,
+.logo-socialmedia-container,
+.links-container,
+.social-media-logo-container {
   display: flex;
   align-items: center;
 }
 
+footer,
+.copyright-disclaimer-container,
+.links-container,
+.footer-bg {
+  flex-direction: column;
+}
+
 .footer-container,
-.footer-copyright-section {
-  width: 100vw;
+.links-container {
+  justify-content: space-between;
 }
 
-.footer-content-section,
-.footer-copyright-infosection {
-  width: 1220px;
+footer {
+  height: 180px;
+  justify-content: center;
+  background: linear-gradient(
+      180deg,
+      rgba(28, 56, 64, 0.7) 0%,
+      rgba(28, 65, 64, 0.8) 0.01%,
+      rgba(7, 27, 32, 0.5) 100%
+    ),
+    url('../../../public/assets/images/concrete-bw.png');
+  background-repeat: repeat;
+  mix-blend-mode: normal;
+}
+
+.footer-bg {
+  background-color: rgba(0, 0, 0, 0.3);
+  mix-blend-mode: normal;
 }
 
 .footer-container {
-  background-color: rgba(40, 48, 49);
-  flex-direction: column;
+  width: 1220px;
+  height: 140px;
+  background-color: rgba(245, 245, 245, 0.05);
 }
 
-.footer-content-section {
-  background-color: rgba(0, 0, 0, 0.3);
-  justify-content: space-between;
-  height: 260px;
+.copyright-disclaimer-container {
+  width: 613px;
+  height: 100px;
+  justify-content: center;
+  box-sizing: border-box;
+  border: 2px solid rgba(140, 92, 2, 1);
+  margin-left: 6px;
 }
 
-.footer-copyright-section {
-  background-color: rgba(23, 23, 23, 1);
-  justify-content: center;
-  height: 40px;
+.find-information-container {
+  width: 381px;
+  height: 100px;
 }
 
-.footer-copyright-infosection {
-  font-size: 0.875;
-  color: rgba(255, 255, 255, 1);
-  text-align: right;
+.copyright-disclaimer-text,
+.find-information-text,
+.logo-link-text {
+  font-family: 'Open Sans';
+  color: rgba(245, 245, 245, 1);
 }
 
-.placeholder-container {
-  width: 25rem;
-  height: 3.75rem;
-  border: 0.25rem solid rgba(188, 122, 0, 1);
-  padding: 0.06rem 0.93rem 0.06rem 1.56rem;
+.copyright-disclaimer-text,
+.find-information-text {
+  font-weight: 400;
+  font-size: 1.125rem;
+  padding: 0.625px 0.438rem;
 }
 
-.placeholder-text {
+.links-container {
+  gap: 1.75rem;
```

Comment:
Generally, I wouldn't mix rem and px. Pick one and stick to it for things like spacings. 

Are you aware of how they differ?

Reply:
> Could you include a picture in the PR description of what the old design looks like and what the PR code makes it look like? It's generally a good idea to include such information in a PR, it means I don't have to run the code just to see what it looks like.

I added screenshots of the new design. The previous design wasn't related to our project. We just recreated the one from the Kaospilot website to have at least something :)

Reply:
Yes, I know. I just didn't know that it was a bad practice to use both of the for spacings. Thanks :)